pkg/util/interrupt/child_linux.go

```diff
@@ -24,4 +24,4 @@ import (
 )
 
 // childSignals are the allowed signals that can be sent to children in Unix variant OS's
-var childSignals = []os.Signal{syscall.SIGINT, syscall.SIGTERM, syscall.SIGQUIT}
```

Comment:
https://github.com/kubernetes/kubernetes/blob/master/pkg/util/interrupt/child.go#L25

separate file (though to be honest i have no idea how we have two files in the same package declaring the same variable.  perhaps there is some build magic going on to pick which file to use when building for a particular platform).

Note that for s2i i didn't follow this pattern, not sure if it's necessary or not.  perhaps @smarterclayton can tell us more.